core/src/Kokkos_View.hpp

```diff
@@ -393,8 +394,11 @@ struct MDSpanViewTraits<Traits,
                                          typename Traits::data_type>::type;
   using mdspan_layout_type =
       typename Impl::LayoutFromArrayLayout<typename Traits::array_layout>::type;
-  using mdspan_type =
-      mdspan<typename Traits::value_type, extents_type, mdspan_layout_type>;
+  using accessor_type = Impl::SpaceAwareAccessor<
+      typename Traits::memory_space,
+      Kokkos::default_accessor<typename Traits::value_type>>;
+  using mdspan_type = mdspan<typename Traits::value_type, extents_type,
+                             mdspan_layout_type, accessor_type>;
```

Comment:
Should we comment somewhere that the converting constructor from a mdspan strips out the accessor and just require convertibility to a space-aware default accessor?

Reply:
not sure what you mean, the constructor takes this thing. So all the rules for mdspan conversion apply anyway. View will only do something with it after the user thing was converted to `mdspan_type`